web.js

```diff
@@ -440,10 +440,27 @@ module.exports = function (options) {
           }
 
           function sendreply (result) {
+            var http_status = result.http$
+            delete result.http$
+
+            // send result
             var repl = reply(result)
+
+            // send cookies
             for (var cookie in request.raw.res.cookies) {
               repl.state(cookie, request.raw.res.cookies[cookie])
             }
+
+            if (http_status) {
+              // redirect if necessary
+              if (http_status.redirect) {
+                repl.redirect(http_status.redirect)
```

Comment:
mind renaming from `repl` its confusing with `REPL`.  This should be `response`, which is what hapi returns.